client/src/utils/api.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+export const makeGetRequest = async (slug) => {
+  const apiUrl = process.env.NEXT_PUBLIC_API_URL;
+  const { data } = await axios.get(`${apiUrl}/${slug}`);
```

Comment:
I believe you need a {withCredentials: true} here

Reply:
That'll come with the refactoring right after this sprint ;)

That function isn't used yet

Reply:
ahh thought it was ayyyyt g